src/core/qgsstoredexpressionmanager.h

```diff
@@ -0,0 +1,158 @@
+/***************************************************************************
+                          qgsstoredexpressionmanager.h
+
+ These classes store and control the management and execution of actions
```

Comment:
Instead of putting the nice details up here, but them down in the actual class dox (also, update them to be relevant ;) )